### ISSUE TITLE
metater user notes

### ISSUE THREAD

**Issue by bendichter** [dandi/dandiarchive#790]
* the License (i) icon should offer more helpful hints about the different license types

* provide brief instruction on the top:
	Red is required
	To add funding information, …

* Protocol (i): protocol.io -> protocols.io

**Comment by bendichter** [comment]
The help text on licenses should link this this page: https://creativecommons.org/licenses/

**Comment by bendichter** [comment]
Even on that site though it's not clear what the difference is between CC0 and CC-BY. Is is that CC0 releases all rights, including credit attribution, and puts the work in the public domain? I think it would be unlikely for a scientist to choose this option, and I would expect them all to want at least credit attribution. I'd expect CC-BY to be the most common. Is there any sort of dark pattern we can use to help users not accidentally choose CC0?

**Comment by bendichter** [comment]
Also, the interface allows you to select more than one license at once. Is that intentional? It seems like the compatibility of CC makes that unnecessary.

**Comment by satra** [comment]
all of openneuro is under CC0. so it's not that scientists do not choose CC0, many do and intentionally so. you can still get attribution through citation even when the dandiset is released under CC0.

the issue with multiple licenses has to do with when we get to mixed license dandisets combining assets from differently licensed assets? at present we expect people to choose 1 license. it also does not make sense for most people to choose multiple.

**Comment by bendichter** [comment]
@satra I am no license expert, I'm just going based on the website, but my understanding is that the difference is when using a CC0 license reusers of the data _may_ cite you, but are not required to and can completely neglect to acknowledge the original source of the data. When using CC-BY reusers of the data _must_ cite you or at least acknowledge the original data source in some way. If that understanding is correct, I am confused why scientists would choose to use CC0. Wouldn't they always want recognition for their work?

**Comment by satra** [comment]
> If that understanding is correct, I am confused why scientists would choose to use CC0. Wouldn't they always want recognition for their work?

yes, your understanding is correct. the reason is that there are many who are willing to break out of that standard mold, at least for certain things, as openneuro has demonstrated. 

one could generally ask the question: should there be anything in the public domain? there are a few lenses to see this through. One, the idea of the Commons and [the commoning process](https://en.wikipedia.org/wiki/Commons#Commoning_as_a_process) and the other through [the value of public domain](https://en.wikipedia.org/wiki/Public_domain#Value). Whether or not this should happen relates to a discourse on expected societal considerations and change, and much more than just DANDI itself. However, DANDI, can play a role in the digital commons, and make information available, when possible, as openly as one can, while leaving the determination to the producers of such information.

**Comment by djarecka** [comment]
I understand that we don't want to extend the list of licenses, but should we add any comments?

**Comment by satra** [comment]
i think such information can be on the meditor related web ui page in the handbook. the information icons have already been updated to reflect the minimal information. 

the web ui page should be the location where we provide guidelines for thinking about identifiers and licenses.

**Comment by waxlamp** [comment]
@satra, are you saying you'd like to resolve this issue by adding to the handbook, and then linking the meditor there?

**Comment by satra** [comment]
i believe the schema has already been updated. the key is whether the UI provides more info at the time of selecting a license. as an example create a LICENSE file in a github repo and see what the UI does. this could be the kind of thing we do at dandiset creation time, since we are limiting the set of licenses available to the user.
